packages/stryker/src/config/ConfigReader.ts

```diff
@@ -41,6 +41,11 @@ export default class ConfigReader {
       this.log.warn(`DEPRECATED: please change the config setting 'reporter: ${JSON.stringify(config.reporter)}' into 'reporters: ${JSON.stringify(config.reporters)}'`);
     }
 
+    if (config.timeoutMs) {
+      config.timeoutMS = config.timeoutMs;
+      this.log.warn(`DEPRECATED: please change the config setting 'timeoutMs: ${JSON.stringify(config.timeoutMs)}' into 'timeoutMS: ${JSON.stringify(config.timeoutMS)}'`);
```

Comment:
Since timeoutMs and timeoutMS are both number you can use:
```ts
this.log.warn(`DEPRECATED: please change the config setting 'timeoutMs: ${config.timeoutMs}' into 'timeoutMS: ${config.timeoutMS}'`);
```
Else the log message will be `... config setting 'timeoutMs: "5000"' into ...` instead of `... config setting 'timeoutMs: 5000' into ...`

Reply:
I have checked files by ctrl + f -> search in all folders -> "timeoutMs"

Reply:
Now let's see if this build works, and then i'll change this